hedera-node/src/main/java/com/hedera/services/state/merkle/MerkleToken.java

```diff
@@ -490,8 +490,11 @@ public void setFeeScheduleFrom(CustomFeesOuterClass.CustomFees grpcFeeSchedule)
 		return	grpcFeeSchedule.getCustomFeesList().stream().map(CustomFee::fromGrpc).collect(toList());
 	}
 
-	public List<CustomFee> customFeeSchedule() {
+	public List<CustomFee> getFeeSchedule() {
```

Comment:
Looks like we have duplicate getters here.

Reply:
Oh yeah .. I guess TokenCreate PR has added it. Removed duplicated getter.